src/containers/list.scss

```diff
@@ -78,4 +78,4 @@ $highlight-color: #ffefd4;
 .beer-description {
   @extend %shared-content-styling;
   width: 45%;
-}
+}
```

Comment:
you have deleted newline character.
All files should end with empty enter. 
Make sure you have eslint/prettier configured to print error for this